.github/workflows/publish.yaml

```diff
@@ -0,0 +1,29 @@
+name: Publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-api-doc:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
+
+      - name: Setup Doxygen
+        run: sudo apt install -y doxygen
+
+      - name: Build module API documentation using Doxygen
+        run: |
+            cmake .
+            make module_api
+            sed -e "s%@PROJECT_.\+_DIR@/%%" Doxyfile.API.in > Doxyfile.API
+            doxygen Doxyfile.API
+
+      - name: Publish generated API documentation to GitHub Pages
+        uses: JamesIves/github-pages-deploy-action@4.1.4
```

Comment:
There is v4.4.0: https://github.com/JamesIves/github-pages-deploy-action/releases

Worth to fix the latest version?